content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
+
+#### Metrics
+
+In the Datadog UI, go to the Metrics Summary page **Metrics > Summary** and search for the metric **datadog.agent.started** or the metric **datadog.agent.running**. If these metrics are not visible right away, it may take a few minutes for the Agent to send the data to the Datadog Platform.
+
+Click on either of the metrics and a Metric panel will open to show additional metadata about where these Metrics are pulled from and any associated tags. Since no tags were configured on this host, you should only see the default tags that Datadog assigns to the metrics including `version` and `host`. See the following section on Agent Configuration Files to learn more about how to add tags.
+
+Explore other default metrics such as **ntp.offset** or **system.cpu.idle**
+
+## Agent Configuration Files
+
+The Agent's main configuration file is `datadog.yaml`. The required parameters are your [Datadog API key][15] which is used to associate your Agent's data with your organization and the Datadog site ({{< region-param key="dd_site" code="true" >}}). See the [sample config_template.yaml][21] for all available configuration options.
+
+Once you have the Agent installed, you can adjust the Agent configuration files to take advantage of other Datadog features including Tags
+
+#### Setting Tags through the Agent Configuration file
+
+1. Locate your Agent's [main configuration file][22]. For Ubuntu, the file locations is 
+`/etc/datadog-agent/datadog.yaml`
+
+2. In the `datadog.yaml` file, locate the tags parameter. Host level tags can be set in the `datadog.yaml` configuration to apply tags on all metrics, traces and logs forwarded from this host.
+
+```yaml
+## @param tags  - list of key:value elements - optional	
+## @env DD_TAGS - space separated list of strings - optional
+## List of host tags. Attached in-app to every metric, event, log, trace, and service check emitted by this Agent.
+##
+## This configuration value merges with `DD_EXTRA_TAGS`, allowing some
+## tags to be set in a configuration file (`tags`), and additional tags to be added
+## with an environment variable (`DD_EXTRA_TAGS`).
+##
+## Learn more about tagging: https://docs.datadoghq.com/tagging/
+#
+# tags:
+#   - team:infra
+#   - <TAG_KEY>:<TAG_VALUE>
+```
+
+3. Uncomment the tags parameter and the provided example `team:infra` tag. 
+```yaml
+## @param tags  - list of key:value elements - optional	
+## @env DD_TAGS - space separated list of strings - optional
+## List of host tags. Attached in-app to every metric, event, log, trace, and service check emitted by this Agent.
+##
+## This configuration value merges with `DD_EXTRA_TAGS`, allowing some
+## tags to be set in a configuration file (`tags`), and additional tags to be added
+## with an environment variable (`DD_EXTRA_TAGS`).
+##
+## Learn more about tagging: https://docs.datadoghq.com/tagging/
+#
+tags:
+   - team:infra
+#   - <TAG_KEY>:<TAG_VALUE>
+```
 
```

Comment:
You could have them add their own personal tags here, too, so they get the idea that things they add show up in the product later. Just uncommenting something provided is less impactful.